### DISCUSSION TITLE
Dúvida sobre contrato entre patrocinadora e organização do evento (PyNE) + APyB

### DISCUSSION THREAD

**Discussion by anapaulamendes** [apyb/comunidade#179] [Pergunta]
Pessoal, a Globo irá entrar na nossa cota de patrocínio Opala e irá nos pagar 22.500,00 e já solicitamos a emissão da Nota Fiscal.
Mas o pessoal da Globo fez um contrato para assinarmos também. Solicitei que fosse em meu nome/cpf como representante do evento. Porém eles nos informaram que só podem estabelecer contrato com outro CNPJ, portanto só podem ter a assinatura de outro CNPJ no nosso contrato. O contrato é somente sobre nosso cumprimento com o que foi estabelecido na cota e o pagamento deles para a gente e proteção da marca Globo. Posso enviar o contrato para vocês assinarem?

Como proceder nesses casos?

**Comment by anapaulamendes** [comment]
cc: @anicelysantos @cecivieira @tiidadavena

**Comment by anicelysantos** [comment]
Olá, @anapaulamendes 
Pode nos enviar o contrato para assinatura, mas antes avaliem se o que está em contrato se enquadra com o estabelecido para o patrocínio.

**Reply by anapaulamendes** [reply]
Verifiquei e está com o que estabelecemos. Vou repassar a vocês, muitíssimo obrigada!
Qual o e-mail para que eu envie o contrato?

**Comment by anicelysantos** [comment]
Envia pra esse, por favor: contato@python.org.br

**Reply by anapaulamendes** [reply]
Enviado! Muito obrigadaaa!
